### ISSUE TITLE
Catch metrical errors

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-kosha/kosha#24]
There are multiple error reports by Prof. Isaacson that there are metrical errors which help in identifying spelling errors. Make it a part of error checking software.

Currently there is a crude version of it available at https://github.com/sanskrit-kosha/kosha/blob/master/scripts/metrical_error.py.

Not yet part of regular error checking script at https://github.com/sanskrit-kosha/kosha/blob/master/scripts/error_summary.py 

This issue tracks the integration of metrical errors as part of regular error checking scripts.

**Comment by Harunaga-Isaacson** [comment]
This will be very good.

**Comment by Harunaga-Isaacson** [comment]
I have noticed one more error.
In the line
शङ्गाटकः स्याच्छैवालविशेषे [च] चतुष्पथे ।
the first word is a mistake for
शृङ्गाटकः .

**Comment by Harunaga-Isaacson** [comment]
I beg your pardon; this was meant to go under the
Corrigenda in Dharaṇikośa text 
issue instead. I am sorry that I don't know how to move it, if that is at all possible.
H.I.

**Comment by drdhaval2785** [comment]
http://skrutable.pythonanywhere.com/wholeFile seems to be fine option to upload the file and get the meters identified for the whole file.
Will write some small script to screen out the problematic verses i.e. which are not full matches.

**Comment by drdhaval2785** [comment]
Currently the skrutable does not allow Arya metre and therefore throws up a lot of false alarms.
I have raised an issue at https://github.com/tylergneill/skrutable/issues/3 . 
Once it gets resolved and if @tylergneill decides to implement https://github.com/tylergneill/skrutable/issues/4, this issue should be closable.

**Comment by tylergneill** [comment]
Good catch on the āryā/jāti problem. Skrutable can of course identify jāti verses, but its resplit options do still need some calibration. I think I have a fix already, and I hope to finish implementing it in the next few days. Highlighting the problematic passages (tylergneill/skrutable#4) will also be implemented.

**Comment by drdhaval2785** [comment]
Thanks
